control/tests/test_hotstore.py

```diff
@@ -29,7 +28,8 @@ def setUp(self):
 
         self.H1 = Hotstore(
             video_object=VP,
-            upload_filepath=self.upload_filepath
+            upload_filepath=self.upload_filepath,
+            video_proto=VP
```

Comment:
last kwarg is unnecessary. [Hotstore](https://github.com/edx/edx-video-pipeline/blob/master/control/veda_hotstore.py#L36)  does not expect this.